content/en/blog/sign-verify-plugins-rules/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "How we Sign and Verify Plugins and Rules"
```

Comment:
Overall comment: I think you need an introductory paragraph that describes what this blog is about and why folks should care. 

I think you could move the general information on SSC attacks up here to the top to set context for the blog. 

Software supply chain attacks are on the rise, especially among open source projects. Over the past few years we've seen an increasing number of attacks, offering proof that attackers find this a viable attack vector. 

To help thwart these threats inside of Falco's own software supply chain we've introduced new processes to sign and verify Falco plugins. In this blog we'll walk you through how we implemented those changes. Our hope is by sharing information about our journey, you can implement similar changes to help secure your own software supply chains. 

Something like that.